webapi/Models/Response/StepwiseThoughtProcess.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json.Serialization;
+
+namespace CopilotChat.WebApi.Models.Response;
+
+/// <summary>
+/// Information about a pass through stepwise planner.
+/// </summary>
+public class StepwiseThoughtProcess
+{
+    /// <summary>
+    /// Steps taken execution stat.
+    /// </summary>
+    [JsonPropertyName("stepsTaken")]
+    public string StepsTaken { get; set; } = string.Empty;
+
+    /// <summary>
+    /// Time taken to fulfil the goal.
+    /// </summary>
+    [JsonPropertyName("timeTaken")]
+    public string TimeTaken { get; set; } = string.Empty;
```

Comment:
nit: specify the unit in the comment since this is a string

Reply:
Added format definition in comment